internal/saucecloud/cypress.go

```diff
@@ -88,9 +90,17 @@ func (r *CypressRunner) runSuites(fileURI string) bool {
 	}
 	defer close(results)
 
+	suites := r.Project.GetSuites()
+	if r.Project.GetSauceCfg().LaunchBy != "" {
+		history, err := r.getTestHistory()
+		if err != nil {
+			return false
+		}
```

Comment:
I don't think we should error the entire run, just because the job order could not be determined.

Reply:
ok. I'll just log the error in the debugging info